### ISSUE TITLE
Client log being spammed with messages

### ISSUE THREAD

**Issue by The-Flying-Guy** [Mithion/ArsMagica2#500]
"Ars Magica 2 >> Compendium unlock state not found to load. Assuming it hasn't been created yet." keeps appaering in the client log, about 6-8x per second, causing significant loss of FPS and well, pointless checking of that state.

**Comment by Robijnvogel** [comment]
Sorry for necro-ing, but it may be worth while to reopen this issue or another issue.

While I was connected to a game on a hosted Multicraft server, my log was constantly spamming that message. Only after I actually created the compendium and looked in it, the compendium unlock file was created on my client for that world and the console/log stopped spamming.
